DSCResources/MSFT_xDnsServerPrimaryZone/MSFT_xDnsServerPrimaryZone.schema.mof

```diff
@@ -0,0 +1,10 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xDnsServerPrimaryZone")]
+class MSFT_xDnsServerPrimaryZone : OMI_BaseResource
+{
+    [Key, Description("DNS Server primary zone name")] String Name;
+    [Required, Description("DNS Server primary zone file")] String ZoneFile;
+    [Write, Description("Dynamic zone update option"), ValueMap{"None","NonsecureAndSecure"}, Values{"None","NonsecureAndSecure"}] String DynamicUpdate;
```

Comment:
This is missing **Secure** value in the valueMap. Is that by design (will implement later) or an oversight?